_includes/section-title-box.html

```diff
@@ -3,6 +3,6 @@
 <header class="va-rr-sg-header va-rr-bg-{{templateclass}}">
   <h1 class="va-rr-sg-title">{{page.title}}</h1>
   {% if page.description %}
-    <p class="va-rr-sg-description">{{page.description}}</p>
+   <div class="usa-content">{{page.description}}</div>
```

Comment:
Instead of replacing `va-rr-sg-description` here, keep it, and just update the styles for that rule. Because the `usa-content` class is part of a library, changing the styles here might cause changes in unexpected places.